research/delf/delf/python/training/global_features_utils.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2021 The TensorFlow Authors All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utilities for the global model training."""
+
+import os
+
+from absl import flags
+from absl import logging
+
+import tensorflow as tf
+import numpy as np
+
+from tensorboard import program
```

Comment:
oh, this one should be considered too, put it before the TF import. (alphabetical ordering)